services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_utils.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from contextlib import asynccontextmanager
+from typing import Any, AsyncGenerator, Dict
+
+import aiodocker
+from aiodocker.utils import clean_filters
+from fastapi import HTTPException, status
+
+logger = logging.getLogger(__name__)
+
+
+@asynccontextmanager
+async def docker_client() -> AsyncGenerator[aiodocker.Docker, None]:
```

Comment:
We have this function 5 times in this repository. I wonder if we could put it in some package like servicelib?

Reply:
Not sure about the others but this one is slightly different from the rest. I don't know if we should bother with that.

Reply:
Not sure? I guess we both have access to the same codebase, right :-) Please verify and create this separate PR if necessary.